src/main/java/nextstep/subway/applicaion/dto/PathRequest.java

```diff
@@ -1,19 +1,30 @@
 package nextstep.subway.applicaion.dto;
 
+import javax.validation.constraints.NotBlank;
 import javax.validation.constraints.NotNull;
+import nextstep.subway.domain.PathSearchType;
 
 public class PathRequest {
     @NotNull
     private Long source;
     @NotNull
     private Long target;
+    private PathSearchType type;
 
     public PathRequest() {
+
     }
 
     public PathRequest(Long source, Long target) {
         this.source = source;
         this.target = target;
+        type = PathSearchType.DISTANCE;
```

Comment:
주 생성자를 활용해 보셔도 좋을 것 같아요 😃 
```suggestion
        this(source, target, PathSearchType.DISTANCE);
```